tracer/samples/RabbitMQ.DistributedTracing/Send/Send.cs

```diff
@@ -18,53 +18,26 @@ public static void Main(string[] args)
-                        string samplingPriority = scope.Span.GetTag(Tags.SamplingPriority);
-
-                        // Add properties to the Headers dictionary in the following way:
-                        //  - "x-datadog-trace-id": "<trace_id>"
-                        //  - "x-datadog-parent-id": "<span_id>"
-                        //  - "x-datadog-sampling-priority": "<sampling_priority>"
-                        properties.Headers.Add(HttpHeaderNames.TraceId, BitConverter.GetBytes(traceId));
-                        properties.Headers.Add(HttpHeaderNames.ParentId, BitConverter.GetBytes(spanId));
-                        properties.Headers.Add(HttpHeaderNames.SamplingPriority, Encoding.UTF8.GetBytes(samplingPriority));
-
-                        // Publish message
-                        channel.BasicPublish(exchange: "",
-                                             routingKey: "hello",
-                                             basicProperties: properties, // Pass the properties with the message
-                                             body: body);
-
-                        // Log message and properties to screen
-                        Console.WriteLine(" [x] Sent {0}", message);
-                        Console.WriteLine("     {0}:{1}", HttpHeaderNames.TraceId, traceId);
-                        Console.WriteLine("     {0}:{1}", HttpHeaderNames.ParentId, spanId);
-                        Console.WriteLine("     {0}:{1}", HttpHeaderNames.SamplingPriority, samplingPriority);
-
-                        // Set Datadog tags
-                        var span = scope.Span;
-                        span.ResourceName = "basic.publish";
-                        span.SetTag(Tags.SpanKind, SpanKinds.Producer);
-                        span.SetTag("amqp.exchange", "");
-                        span.SetTag("amqp.routing_key", "hello");
-                    }
+                                            durable: false,
+                                            exclusive: false,
+                                            autoDelete: false,
+                                            arguments: null);
+
+                    string message = "Hello World!";
+                    var body = Encoding.UTF8.GetBytes(message);
+
+                    // Create BasicProperties and a Headers dictionary to store header information
+                    var properties = channel.CreateBasicProperties();
+                    properties.Headers = new Dictionary<string, object>();
+
+                    // Publish message
+                    channel.BasicPublish(exchange: "",
+                                            routingKey: "hello",
+                                            basicProperties: properties, // Pass the properties with the message
+                                            body: body);
+
+                    // Log message and properties to screen
+                    Console.WriteLine(" [x] Sent {0}", message);
```

Comment:
I think we should demonstrate that the trace ID at least is correctly propagated in these samples, so I would create a manual span around the publish, and write the trace ID to the console

Reply:
I think that's a good idea. Though, I also don't want people to believe they have to add a custom span on the producer side. But, a good comment will do the trick.
I'll add that, and will update the csproj and UpdateVersions then

Reply:
done in 23ce9c689